server/aws/nodes.yaml

```diff
@@ -354,6 +393,11 @@ Resources:
                 BRANCH="${BRANCH}"
                 DAEMONOPTS="${DAEMONOPTS}"
                 -i -u ubuntu
+            01_installUploadCron:
+              command: >
+                hour=$(($(dd if=/dev/urandom count=1 2>/dev/null | cksum | cut -d' ' -f1) % 24)) &&
+                minute=$(($(dd if=/dev/urandom count=1 2>/dev/null | cksum | cut -d' ' -f1) % 60)) &&
+                ( crontab -u ubuntu -l; echo "${minute} ${hour} * * * /home/ubuntu/upload_lmdb.sh 1>>/home/ubuntu/upload_lmdb.log 2>&1"; ) | crontab -u ubuntu -
```

Comment:
The `hour` and `minute` times are chosen at random to avoid nodes all being down at the same time. In the unlikely event that two times are chosen close to one another, it is very simple to log into the instance and manually change the crontab (and offset it by 12 hours).